rust/kernel/cpu.rs

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: GPL-2.0
+
+//! CPU utilities for SMP systems.
+//!
```

Comment:
If you look at other module, we tend to refer to the C header file here. In this case, you probably want to mention `smp.h`.